docs/overview-genieve.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+---
+
+# About To-Do service REST API
+
+The To-Do service offers a cloud-hosted task list that allows users to post tasks and receive reminders. This user-friendly, lightweight, and reliable cloud service is perfect for app developers who want to create reminder applications.
+
+Users can register and update their profiles in under a minute. Each task will include crucial properties like the title, description, and due date.
+
+Developers can customize the To-Do service with their branding and incorporate features for seamless integration with other services. Discover how our service can work with various apps to facilitate automated task reminders.
+
+## Get started with To-Do service
+
+Learn how to manage the two key resources: **users** and **tasks**. The following tutorials will help you get up to speed with the To-DO service REST API.
+
+**NOTE:** Before you get started, set up your development system for these tutorials. You only have to do this one time per development system.
+
+* [Before you start a tutorial](tutorials/before-you-start-a-tutorial)
+
+After your system is ready, these tutorials show you how to perform common tasks.
+
```

Comment:
For an overview, some structure or context to these links would help a new reader who is unfamiliar with the product.